TwilioVerifySDK/TwilioSecurity/Sources/Biometrics/BiometricError.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  BiometricError.swift
+//  TwilioVerifySDK
+//
+//  Created by Alejandro Orozco Builes on 17/09/21.
+//  Copyright © 2021 Twilio. All rights reserved.
+//
+
+import Foundation
+import LocalAuthentication
+
+public enum BiometricError: Error, LocalizedError {
+
+    // MARK: - Biometric LAError cases
+
+    case appCancel
+    case authenticationFailed
+    case invalidContext
+    case notInteractive
+    case passcodeNotSet
+    case systemCancel
+    case userCancel
+    case userFallback
+    case biometryLockout
+    case biometryNotAvailable
+    case biometryNotEnrolled
+
+    public var errorDescription: String? {
+        return "Biometric error: LAError \(String(describing: self))"
+    }
+
+    // MARK: - Resolution
+
+    public static func given(_ error: NSError?) -> Self? {
```

Comment:
<table data-meta="generated_by_UnitTests">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/twilio/twilio-verify-ios/blob/84517577f916f06073de7ffb2ba09f81b7655c5d/TwilioVerifySDK/TwilioSecurity/Sources/Biometrics/BiometricError.swift#L34"></span>Function should have complexity 10 or less: currently complexity equals 13<br />
<code>cyclomatic_complexity</code> <code>BiometricError.swift:34</code></td>
    </tr>
  </tbody>
</table>

Reply:
Removed complexity verification for Switch statement.